pydantic_xml/serializers.py

```diff
@@ -558,6 +558,63 @@ class HomogeneousSerializerFactory:
     Homogeneous collection type serializer factory.
     """
 
+    class TextSerializer(Serializer):
+        def __init__(
+                self, model: Type['pxml.BaseXmlModel'], model_field: pd.fields.ModelField, ctx: Serializer.Context,
+        ):
+            assert len(model_field.sub_fields) == 1
+            if isclass(model_field.type_) and issubclass(model_field.type_, pxml.BaseXmlModel):
+                raise errors.ModelFieldError(
+                     model.__name__, model_field.name, "Inline list value should be of scalar type",
+                )
+
+        def serialize(
+                self, element: etree.Element, value: Collection[Any], *, encoder: XmlEncoder, skip_empty: bool = False,
+        ) -> Optional[etree.Element]:
+            if value is None or skip_empty and len(value) == 0:
+                return element
+
+            encoded = " ".join(encoder.encode(val) for val in value)
+            element.text = encoded
+            return element
+
+        def deserialize(self, element: etree.Element) -> Optional[List[Any]]:
+            if element.text is None:
+                return []
+
+            return [value for value in element.text.split()]
+
+    class AttributeSerializer(Serializer):
+        def __init__(
+                self, model: Type['pxml.BaseXmlModel'], model_field: pd.fields.ModelField, ctx: Serializer.Context,
+        ):
+            assert len(model_field.sub_fields) == 1
+            if issubclass(model_field.type_, pxml.BaseXmlModel):
+                raise errors.ModelFieldError(
+                     model.__name__, model_field.name, "Inline list value should be of scalar type",
+                )
+
+            ns_attrs = model.__xml_ns_attrs__
+            name = ctx.entity_name or model_field.alias
+            ns = ctx.entity_ns or (ctx.parent_ns if ns_attrs else None)
+            nsmap = ctx.parent_nsmap
+            
+            self.attr_name = QName.from_alias(tag=name, ns=ns, nsmap=nsmap, is_attr=True).uri
+
+
```

Comment:
too many blank lines. Please install pre-commit hooks to check codestyle before commit.

Reply:
Got pre-commit running now, it should be fixed.